### DISCUSSION TITLE
The solution is not ready to run out of the box

### DISCUSSION THREAD

**Discussion by ArvindRSingh** [ASP-NET-MVC-Boilerplate/Framework#86] [Q&A]
On opening the Framework.sln no project was loaded and on trying to reload it says 

> BoxedMapper\Source\Boxed.Mapping\Boxed.Mapping.csproj : error  : The project file cannot be opened by the project system, because it is missing some critical imports or the referenced SDK cannot be found.
> 
> Detailed Information:
> Unable to locate the .NET Core SDK. Check that it is installed and that the version specified in global.json (if any) matches the installed version.

I have .NET core SDKs 1.0, 1.1, 2.0, 2.1, 2.2 and working for other projects.

**Comment by RehanSaeed** [comment]
You must have 2.2.300 of the SDK installed. See the `global.json` file.
